runtimes/ruby-3.1/README.md

```diff
@@ -18,7 +18,7 @@ printf 'def main(request, response)\n    return response.json({:n => rand()})\ne
 2. Build the code:
 
 ```bash
-docker run --rm --interactive --tty --volume $PWD:/usr/code openruntimes/ruby:3.1 sh /usr/local/src/build.sh
+docker run -e ENTRYPOINT_NAME=index.rb --rm --interactive --tty --volume $PWD:/usr/code openruntimes/ruby:3.1 sh /usr/local/src/build.sh
```

Comment:
Why do we need a `ENTRYPOINT_NAME` 

1. build.sh does not seem to be using this env variable anywhere
2. Didn't we rename it to `INTERNAL_RUNTIME_ENTRYPOINT`  ??

Please check this in the other files as well.

Reply:
We need `ENTRYPOINT_NAME` for building, and `INTERNAL_RUNTIME_ENTRYPOINT` for running.

It is not needed in Ruby, but it is needed in Rust, Dart or Swift. I decided to put it into all of them for consistency. I think OR should be language agnostic, and I think you should know all runtimes once you learn to use one. Even tho it's not needed for Python or Node, it's good practice to learn to use it, otherwise you will be getting weird errors in build of other runtimes and you would not know what is going on.

Reply:
We should keep it only for runtimes that need it. For all other runtimes, it is information that is never used.